pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1268,7 +1268,16 @@ class Timedelta(_Timedelta):
 
             kwargs = {key: _to_py_int_float(kwargs[key]) for key in kwargs}
 
-            nano = convert_to_timedelta64(kwargs.pop('nanoseconds', 0), 'ns')
+            # GH43764, making sure any nanoseconds contributions from any kwarg is taken into consideration
```

Comment:
umm this is just duplicating code from below

Reply:
I don't know exactly which code you're referring to, but if I do not catch all nanoseconds contributions when a `Timedelta` instance is prepared using any of the "microseconds", "milliseconds" and "seconds" kwargs or any combination therefrom, the potential nanosecond contributions from every of those kwargs must be caught here, otherwise it will be lost information, e.g. `Timedelta(seconds=1e-9, milliseconds=1e-5, microseconds=1e-1)`.